diff --git a/dist/index.js b/dist/index.js
index 64ca0123f4086fd4fd99e49cafe7c0604164d4fa..be666e8c0ea8a4daba5b3b651b8cb438702d4373 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -66,7 +66,7 @@ function lookupAdditionalManifestEntries(useInjectManifest, injectManifest, work
   return useInjectManifest ? injectManifest.additionalManifestEntries || [] : workbox.additionalManifestEntries || [];
 }
 function normalizeIconPath(path) {
-  return path.startsWith("/") ? path.substring(1) : path;
+  return path?.startsWith("/") ? path.substring(1) : path;
 }
 function includeIcons(icons, globs) {
   Object.keys(icons).forEach((key) => {
